### PR TITLE
fix(suite-native): correct error for firmwareless device

### LINKED ISSUE #10611: App shows wrong message with unitialized Trezor device
When user connects Trezor device which is not initialized, it shows this message: 
Image

Please replicate if it's really happening

Right behaviour - app should show this message:
https://www.figma.com/file/IidkuNfa5OWtPgta9jF8Gl/T-%E2%80%A2-Handoff?type=design&node-id=4068%3A24715&mode=design&t=ZuRUCmI0UfkzRx3u-1

### PR DESCRIPTION
**Pull request by PeKne**
This commit fixes the error message for firmwareless devices. Until now if you have connected firmwareless device, the error message saying that the device is in bootloader mode was displayed. After this change, correct message instructing user to setup the device is displayed.

Closes #10611

### CODE REVIEW

File: suite-common/wallet-core/src/device/deviceReducer.ts [hunk 1]
```diff
@@ -723,3 +723,9 @@ export const selectDeviceLanguage = (state: DeviceRootState) => {
     const features = selectDeviceFeatures(state);
     return features?.language ?? null;
 };
+
+export const selectHasDeviceFirmwareInstalled = (state: DeviceRootState) => {
+    const device = selectDevice(state);
+
+    return device?.firmware !== 'none';
```
Comment (matejkriz) [inline suite-common/wallet-core/src/device/deviceReducer.ts]: maybe more robust condition to not return `hasDeviceFirmwareInstalled === true` if there is no device?
```suggestion
    return !!device && device.firmware !== 'none';
```

Comment (PeKne) [reply suite-common/wallet-core/src/device/deviceReducer.ts]: makes sense 👍

Comment (PeKne) [reply suite-common/wallet-core/src/device/deviceReducer.ts]: fixed by [ec58dac](https://github.com/trezor/trezor-suite/pull/10635/commits/ec58dac89d70658f4144cb973e486085c6241095)

**Review by matejkriz — APPROVED** [review]
👍

**Pull request merged**